### ISSUE TITLE
Add support for keybind with 3rd - 5th mouse buttons

### ISSUE THREAD

**Issue by simonknittel** [bennieboj/BlishHud-Mounts#18]
It would be cool if you could add support to use the 3rd, 4th or 5th mouse button for the keybind to open the radial.

I'm coming from GW2Radial and until now I used the _forward button_ of my mouse to trigger the radial.

**Comment by bennieboj** [comment]
Thanks for the report.
Mouse buttons are not supported for keybinds in general in Blishhud just yet. It's planned but don't expect it soon.
See https://github.com/blish-hud/Blish-HUD/issues/611

**Comment by simonknittel** [comment]
Oh, ok. Thanks for the hint.

Edit:
I found the following workaround:
Logitech Options (with the MX Master 3) allows you to have mouse mapping profiles for individual applications. I've created a profile for Guild Wars 2 and mapped my forward button to F8 instead of that forward behavior. This way I can use the forward button/F8 for opening the radial. And because of the profile this mapping only has effect in Guild Wars 2. Therefore I can still use the default forward behavior in my browser.
